libr/bin/format/elf/elf.c

```diff
@@ -2371,7 +2371,10 @@ struct Elf_(r_bin_elf_obj_t)* Elf_(r_bin_elf_new_buf)(RBuffer *buf) {
 	struct Elf_(r_bin_elf_obj_t) *bin = R_NEW0 (struct Elf_(r_bin_elf_obj_t));
 	bin->kv = sdb_new0 ();
 	bin->b = r_buf_new ();
-	bin->size = buf->length;
+//	bin->size = buf->length;
+	ut64 curr = lseek (buf->fd, 0, SEEK_CUR);
```

Comment:
Agreed that RBuffer can work without an fd, but in this case, it is called from http://computers.tutsplus.com/tutorials/how-to-ocr-documents-for-free-in-google-drive--cms-20460 which sets this. But as you commented there that we should have independent working of rbuff, in that case, it seems like the current way I am trying to implement won't be doable without the fd since I am trying to do seek+read in the r_buf_fcpy_at part to which only the RBuffer is passed and no other related info about the bin (esp. file size and fd to read from).